daemon/logger/logger.go

```diff
@@ -15,4 +21,6 @@ type Logger interface {
 	Log(*Message) error
 	Name() string
 	Close() error
+	GetReader() (io.Reader, error)
+	LogPath() string
```

Comment:
@LK4D4 ah that comment got lost since I changed the neighbor line. Here's my problem with this:

Why do we surface the LogPath in the `docker inspect` response? What really uses it? It's not even valid for most of the log drivers? The only reason this func exists is, we surface the path of the logfile back to the Remote API (which I don't think there's a good reason for it unless I'm missing a big use case of it).

Is there an option like to put this LogPath field in the Remote API on a deprecation path so that we can get rid of this requirement in the long term? I think for now I'll just remove this from the interface and type-assert in the `container.go#startLogging()` method.